.github/workflows/docs.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
     - main
     - 3.0_develop
+    - doc_week
```

Comment:
@ketan96-m Thanks for fixing the wrong information I originally told you. You're now deploying the docs to https://ssl-hep.github.io/ServiceX_frontend/ on each push to this branch where the docs build successfully first :+1:. Though I would recommend removing this addition to `doc_week` _before PR merge_ so that it doesn't end up being a strange vestigial line that causes personnel confusion later (though it won't cause any problems)